common/service.go

```diff
@@ -132,6 +132,48 @@ type Service interface {
 // It can be used for launching service from tests, etc.
 func InitializeService(service Service, config ServiceConfig) (*RestServiceInfo, error) {
```

Comment:
That's just the context that diff shows. Full  docstring was:

// InitializeService initializes the service with the
// provided config and starts it. The channel returned
// allows the calller to wait for a message from the running
// service. Messages are of type ServiceMessage above.
// It can be used for launching service from tests, etc.
func InitializeService(service Service, config ServiceConfig) (*RestServiceInfo, error) {